lib/templates.js

```diff
@@ -17,16 +18,29 @@ const tags = ['<%=', '=%>'];
  * hash or it will **not** raise an error if the value is missing.
  * @param {string} path - path to the file which will be processed.
  **/
-const fillInTemplateValues = (path, templateData) => {
-  const fileBody = fs.readFileSync(path, 'utf8');
-  try {
-    return mustache.render(fileBody, templateData, {}, tags);
-  } catch (err) {
-    logger.Debug(err);
-    return '';
+const fillInTemplateValues = (filePath, templateData) => {
+  const fileBody = fs.readFileSync(filePath, 'utf8');
+  if (qualifedForTemplateProcessing(filePath)) {
+    logger.Debug('Processing module file as a template');
+    try {
+      return mustache.render(fileBody, templateData, {}, tags);
+    } catch (err) {
+      logger.Debug(err);
+      return '';
+    }
+  } else {
+    logger.Debug('Read Body');
+    return fs.createReadStream(filePath);
   }
 };
 
+const qualifedForTemplateProcessing = (filePath) => {
+  allowedTemplateExtentions = ['.css', '.js', '.liquid', '.yml', '.xml', '.json', '.svg', '.graphql']
+  if (allowedTemplateExtentions.includes(path.extname(filePath))) {
+    return true
+  }
```

Comment:
You don't need the `if` here, right? `return llowedTemplateExtentions.includes(path.extname(filePath))` will do